packages/open-api-gateway/README.md

```diff
@@ -618,6 +618,20 @@ Authorizers.custom({
 });
 ```
 
+#### Integrations
+
+Integrations are used by API Gateway to service requests.
+
+##### Lambda Integration
+
+Currently, the only built-in integration is a lambda integration. You can construct one using `Integrations.lambda(yourLambdaFunction)`.
+
+##### Custom Integrations
+
+You can implement your own integrations by inheriting the `Integration` class and implementing its `render` method. This method is responsible for returning a snippet of OpenAPI which will be added as the `x-amazon-apigateway-integration` for an operation. Please refer to the [API Gateway Swagger Extensions documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions-integration.html) for more details.
+
+You can also optionally override the `grant` method if you need to use CDK to grant API Gateway access to invoke your integration.
```

Comment:
Wouldn't you always want to do this?

Reply:
It depends - you might be proxying to some third party non-aws http endpoint in which case there's no iam permissions etc to set up :)

Reply:
Makes sense.